tests/test_dask_distance.py

```diff
@@ -172,6 +197,19 @@ def test_2d_pdist(metric, kw, seed, u_shape, u_chunks):
     a_u = 2 * np.random.random(u_shape) - 1
     d_u = da.from_array(a_u, chunks=u_chunks)
 
+    if metric == "mahalanobis":
+        if "VI" not in kw:
+            kw["VI"] = 2 * np.random.random(2 * u_shape[-1:]) - 1
+        elif kw["VI"] is None:
+            kw.pop("VI")
+    elif metric == "seuclidean":
+        if "V" not in kw:
+            kw["V"] = 2 * np.random.random(u_shape[-1:]) - 1
+        elif kw["V"] is None:
+            kw.pop("V")
+    elif metric == "wminkowski":
+        kw["w"] = np.random.random(u_shape[-1:])
```

Comment:
Were unable to test negative values as well due to issue ( https://github.com/scipy/scipy/issues/7953 ).